sdk/src/main/java/com/hedera/hashgraph/sdk/HederaTrustManager.java

```diff
@@ -13,23 +14,12 @@
     @Nullable
     private final String certHash;
 
-    HederaTrustManager(@Nullable NetworkName networkName, AccountId nodeAccountId) {
-        if (networkName == null) {
+    HederaTrustManager(@Nullable ByteString certHash) {
+        if (certHash == null) {
             this.certHash = null;
-            return;
-        }
-
-        var nodeAddressBook = Network.nodeAddressBooks.get(networkName);
-        if (nodeAddressBook == null) {
-            throw new IllegalStateException("Unrecognized network name");
+        } else {
```

Comment:
nit: Can be simplified
```suggestion
        if (certHash != null) {
```